eth-contracts/scripts/truffle-test.sh

```diff
@@ -34,7 +34,7 @@ docker run --name audius_ganache_cli_eth_contracts_test -d -p 8556:8545 truffles
 
 # compile and lint
 ./node_modules/.bin/truffle compile
-./scripts/lint.sh
+# ./scripts/lint.sh
```

Comment:
we do want this

Reply:
yeah ofc, pushed this accidentally

Reply:
also this PR is only for merge into my other working branch so it's fine. will clean all this up before taking to master